nixos/release.nix

```diff
@@ -149,9 +149,9 @@ in rec {
     inherit system;
   });
 
-  iso_graphical = forMatchingSystems [ "x86_64-linux" ] (system: makeIso {
-    module = ./modules/installer/cd-dvd/installation-cd-graphical-kde.nix;
-    type = "graphical";
+  iso_graphical_plasma5 = forMatchingSystems [ "x86_64-linux" ] (system: makeIso {
```

Comment:
For brevity, maybe call it `iso_plasma5`? "Graphical" is implied by Plasma.

Reply:
Yeah that is better :+1: